services/inspection-report/src/components/View/index.js

```diff
@@ -9,27 +9,41 @@ import ScrollToTop from 'components/ScrollToTop';
  * @param description {string}
  * @param title {string}
  * @param viewName {string}
+ * @param style {CSSProperties}
+ * @param className {string}
  * @returns {JSX.Element}
  * @constructor
  */
-export default function View({ children, description, title, viewName }) {
+export default function View({
+  children,
+  description,
+  title,
+  viewName,
+  style,
+  className,
```

Comment:
Prefer the destructuring operator like `...passThroughProps ` and `<div {...passThroughProps}>`